.github/workflows/maven-service/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Maven package
+on: [workflow_call, workflow_dispatch]
+
+jobs:
+  publish-maven:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      packages: write
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      - name: Publish maven-service package
+        run: cd ${{ github.workspace }}/maven-service && mvn --batch-mode deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we have documentation on how to add this token, and whose token it is (is this token "your" token?)

Reply:
This token is made available to workflows automatically by GitHub.